dss/storage/validator.py

```diff
@@ -44,6 +44,7 @@ def remove_json_fields(json_data: dict, path: List[str], fields: List[str]):
 )
 
 resolver = validators.RefResolver(referrer='', base_uri='')
+EXTRA_FIELD_ERROR_REGEX = re.compile(r"(?<!regexes: )'([^']+)',?")
```

Comment:
This is the regex i'm using to extract the fields `r"(?<!regexes: )'([^']+)',?"`. If only only alphanumeric fields are allowed I could simplify it. Though there is the possibility the pattern defined in `patternProperty` could still match as a field.